doc/api-deprecated.apib

```diff
@@ -0,0 +1,558 @@
+                    "execution_id": "styx-run-6b1962c6-23ba-4245-a13b-ec3baa4b21d5",
+                    "last_exit": 20,
+                    "messages": [
+                        {
+                            "level": "WARNING",
+                            "line": "Exit code: 20"
+                        }
+                    ],
+                    "retry_cost": 0.2,
+                    "retry_delay_millis": 600000,
+                    "tries": 2,
+                    "trigger": {
+                        "@type": "natural"
+                    },
+                    "trigger_id": "natural-trigger"
+                }
+            }]
+        }
+
+## Events [/{version}/cli/events/{component_id}/{endpoint_id}/{instance}]
+
++ Parameters
+    + version: `v0` (enum[string]) - API version
+        + Members
+            + `v0`
+            + `v1`
+    + component_id: `styx-canary` (string) - Workflow Component
+    + endpoint_id: `StyxCanary` (string) - Endpoint ID
+    + instance: `2017-01-01T01` (string) - Worfklow Instance parameter
+
+### Get Workflow Instance events [GET]
+
++ Response 200 (application/json)
+
+        {
+            "events": [{
+                "event": {
+                    "@type": "triggerExecution",
+                    "trigger": {
+                        "@type": "natural"
+                    },
+                    "workflow_instance": "styx-canary/StyxCanary/2017-01-01T01"
+                },
+                "timestamp": 1483228800000
+            },
+            {
+                "event": {
+                    "@type": "dequeue",
+                    "workflow_instance": "styx-canary/StyxCanary/2017-01-01T01"
+                },
+                "timestamp": 1483228800005
+            }]
+        }
+
+## Events [/{version}/cli/events]
+
++ Parameters
+    + version: `v0` (enum[string]) - API version
+        + Members
+            + `v0`
+            + `v1`
+
+### Inject Event [POST]
+
++ Request (application/json)
+
+        {
+            "@type": "dequeue",
+            "workflow_instance": "styx-canary/StyxCanary/2017-01-01T01"
+        }
+
++ Response 200 (application/json)
+
+        {
+            "@type": "dequeue",
+            "workflow_instance": "styx-canary/StyxCanary/2017-01-01T01"
+        }
+
+## Trigger [/{version}/cli/trigger]
+
++ Parameters
+    + version: `v0` (enum[string]) - API version
+        + Members
+            + `v0`
+            + `v1`
+
+### Trigger Workflow Instance [POST]
+
++ Request (application/json)
+
+        {
+            "parameter": "2017-01-01T01",
+            "workflow_id": {
+                "component_id": "styx-canary",
+                "endpoint_id": "StyxCanary"
+            }
+        }
+
++ Response 200 (application/json)
+
+        {
+            "parameter": "2017-01-01T01",
+            "workflow_id": {
+                "component_id": "styx-canary",
+                "endpoint_id": "StyxCanary"
+            }
+        }
+
+# Group Global Config
```

Comment:
Actually should we rename this `Group Styx Config` to reflect the class name as in all the other cases? Same in the file for APIs version 2. I am not sure how I came up with this "Global" name before..